api/validator.js

```diff
@@ -44,7 +45,8 @@ const SCHEMA_OUTPUT = Joi.object()
     .keys({
         jobs: SCHEMA_JOBS,
         workflow: Workflow.workflow,
-        errors: Joi.array().items(Joi.string()).optional()
+        errors: Joi.array().items(Joi.string()).optional(),
+        annotations: Annotations.annotations
```

Comment:
@r3rastogi  Sorry, I thought you wouldn't need in pipeline. Can you add it back? :p 
You still need it in `SCHEMA_JOB_PERMUTATION` though.